static/styles/dark_theme.css

```diff
@@ -538,6 +539,7 @@ body.dark-theme {
     .all-messages-sidebar-menu-icon:hover,
     .starred-messages-sidebar-menu-icon:hover,
     .stream-sidebar-menu-icon:hover,
+    .drafts-sidebar-menu-icon:hover,
```

Comment:
The hover state on the menu Icon of the drafts section was not working as intended compared to the All messages and stared massage section on the left sidebar, this caused an issue of UI inconsistency on the draft message section.

This commit fixed this by adding the class associated with the drafts section to the CSS code on the menu-icons code block in dark_theme.css as it was missing that line of code.